test/podFailover/workload/scenarios/sameNodeFailover.yaml

```diff
@@ -14,7 +14,7 @@ namespace: samenode-pod-failover-1pod1pvc
 
 podCount: 3
 
-pvcCount: 3
+pvcCount: 1
```

Comment:
Is it pvcPerPod? If that is the case, we could change the name to pvcPerPodCount in order to avoid confusion.